tests/test_de_serialisation.py

```diff
@@ -133,9 +167,7 @@ def test_serialize_deserialise():
         ),
         "rank": LabelTransformer(nullable=False),
         "job": DropTransformer(),
-        # "date": ChainTransformer(
-        #     [DateTimeTransformer(), MinMaxTransformer(nullable=False)]
-        # ),
+        "date": DateTimeTransformer(epoch="1993-06-04"),
```

Comment:
not sure to fully understand the epoch. For what i understand, its the date of reference during the transformation from date to float (nb of days since reference - 1970-01-01 being 0 by default). Is it only testing this aspect?

Reply:
Here it is testing that the serialiser works also when given another start date of reference, so from what I understand in this case 0 would start the "1993-06-04". 
But I am just testing  the de/serialiser works not the underlying library